test/Microsoft.AspNet.Mvc.Core.Test/Internal/UrlUtilityTests.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Core.Internal;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core.Test.Internal
```

Comment:
`Microsoft.AspNet.Mvc.Internal`

Test code is always in the same namespace as product code.